### ISSUE TITLE
`comma-dangle` `always-multiline` triggers no warning

### ISSUE THREAD

**Issue by pascalduez** [eslint/eslint#2406]
```js
var foo = {
  bar: "baz",
  qux: "quux"
};

var foo = { bar: "baz", qux: "quux", };
``` 

``` yaml
#.eslintrc
rules:
  comma-dangle: "always-multiline"

``` 

```json
  "version": "0.20.0",
```

Result: no warning returned.

**Comment by gyandeeps** [comment]
I just tried your example on the [Demo](http://eslint.org/demo/), and i was able to see the expected results. Give it a shot.

```js
/* eslint comma-dangle:[2, "always-multiline"] */
var foo = {
  bar: "baz",
  qux: "quux"
};

var foo = { bar: "baz", qux: "quux", };
```

**Comment by kentor** [comment]
.eslintrc should be
```yaml
  comma-dangle: [2, "always-multiline"]
```

**Comment by gyandeeps** [comment]
Yes.

**Comment by pascalduez** [comment]
Oh dang, sorry guys. I got a bit confused by the docs I guess.
